tably.py

```diff
@@ -163,9 +218,11 @@ def format_alignment(align, length):
         return '{:c<{l}.{l}}'.format(align, l=length)
 
 
-def get_units(units):
+def get_units(units, no_escape):
```

Comment:
Well, it's true that `get_units` is not called outside the class. But, instead of directly make it a class method, I am thinking about doing something different in the `__init__` method of `Tably` class to avoid passing `no_escape` and doing if-else branching inside the methods every time they are called.